parm/config/gefs/config.resources

```diff
@@ -293,6 +293,18 @@ case ${step} in
     export NTASKS=${npe_wavepostpnt}
     ;;
 
+  "extractvars")
+    export wtime_extractvars="00:30:00"
+    export npe_extractvars=6
```

Comment:
I have been considering turning this into an mpmd job, but this depends on the time it will take to run this job when there are 48 lead days to process.

Reply:
I have set `npe_extractvars` to `1`. Might need to revisit this if an MPMD job is needed.